client/my-sites/checkout/checkout-thank-you/akismet-checkout-thank-you.tsx

```diff
@@ -87,4 +95,56 @@ const AkismetCheckoutThankYou: FunctionComponent< AkismetCheckoutThankYouProps >
 	);
 };
 
+function ThankYouAPIKeyClipboard() {
+	const translate = useTranslate();
+	const [ isCopied, setIsCopied ] = useState( false );
+	const { data, isError, isLoading } = useAkismetKeyQuery();
+
+	useEffect( () => {
+		if ( isCopied ) {
+			const confirmationTimeout = setTimeout( () => setIsCopied( false ), 4000 );
+			return () => clearTimeout( confirmationTimeout );
+		}
+	}, [ isCopied ] );
+
+	const showConfirmation = () => {
+		setIsCopied( true );
+	};
+
+	if ( isError || isLoading ) {
+		return null;
+	}
+
+	const akismetApiKey = data ?? '';
+	const keyInputSize = akismetApiKey ? akismetApiKey.length + 5 : 0;
+
+	return (
+		<>
+			<p className="akismet-checkout-thank-you__api-key-header">
+				{ __( 'Your Akismet API Key is:', 'akismet-thank-you' ) }
+			</p>
+			<div
+				className={ classNames(
+					'akismet-checkout-thank-you__key-clipboard',
+					isLoading ? ' loading' : ''
```

Comment:
I don't think it makes a difference but you shouldn't need the space here anymore 😄 
```suggestion
					isLoading ? 'loading' : ''
```

Reply:
yep yep - applied!